discord/cog.py

```diff
@@ -693,7 +694,11 @@ def _load_from_module_spec(self, spec: importlib.machinery.ModuleSpec, key: str)
             raise errors.NoEntryPointError(key)
 
         try:
-            setup(self)
+            if inspect.iscoroutinefunction(setup):
+                create_task(setup(self))
```

Comment:
Aren't you supposed to store the created task then eventually await for it?

Reply:
pretty sure not.

![image](https://user-images.githubusercontent.com/70831061/160220131-23c2bc6e-ac1b-4823-93d2-28e670fcfd94.png)